server/src/main/java/org/apache/druid/metadata/SegmentsMetadata.java

```diff
@@ -33,11 +33,11 @@
 import java.util.Set;
 
 /**
- * The difference between this class and org.apache.druid.sql.calcite.schema.MetadataSegmentView is that this
- * class resides in Coordinator's memory, while org.apache.druid.sql.calcite.schema.MetadataSegmentView resides
- * in Broker's memory.
+ * The difference between this class and org.apache.druid.sql.calcite.schema.MetadataSegmentView is that this class
+ * resides in Coordinator's memory, while org.apache.druid.sql.calcite.schema.MetadataSegmentView resides in Broker's
+ * memory.
  */
-public interface MetadataSegmentManager
+public interface SegmentsMetadata
```

Comment:
same as https://github.com/apache/incubator-druid/pull/7306#discussion_r315040784

Reply:
Renamed to `SegmentsMetadataManager` 👍